mono/utils/mono-signal-handler.h

```diff
@@ -10,6 +10,7 @@
 #define __MONO_SIGNAL_HANDLER_H__
 
 #include "config.h"
+#include "glib.h"
```

Comment:
Usually `config.h` goes in the `.c` file.   And `#include <glib.h>` is what we usually write.  But also I'm not sure why glib is needed in this header

Reply:
I didn't add config.h there - would not want to mess with that in this PR. glib is needed because of the use of `gboolean`, but it looks like so far everyone was incidentally including glib.h before including this file.